test/test_helper.rb

```diff
@@ -1,5 +1,5 @@
+require 'simplecov'
+SimpleCov.start
```

Comment:
haha this needs to be at the top, that's why our coverage was so low at first

Reply:
good catch!